cns/restserver/ipam.go

```diff
@@ -105,7 +185,7 @@ func (service *HTTPRestService) setIPConfigAsAvailable(ipconfig *cns.ContainerIP
 	ipconfig.State = cns.Available
 	ipconfig.OrchestratorContext = nil
 	service.PodIPConfigState[ipconfig.ID] = ipconfig
-	service.PodIPIDByOrchestratorContext[podInfo.GetOrchestratorContextKey()] = ""
+	delete(service.PodIPIDByOrchestratorContext, podInfo.GetOrchestratorContextKey())
```

Comment:
Also not for this particular line, but below in ReleaseIpConfig, the error returned is "Pod->IPIP exists..." when I think you meant "Pod->IPID exists..."

Reply:
Updated comment